PhysicsTools/PatAlgos/python/slimming/metFilterPaths_cff.py

```diff
@@ -23,6 +23,7 @@
 Flag_hcalLaserEventFilter = cms.Path(hcalLaserEventFilter)
 Flag_EcalDeadCellTriggerPrimitiveFilter = cms.Path(EcalDeadCellTriggerPrimitiveFilter)
 Flag_EcalDeadCellBoundaryEnergyFilter = cms.Path(EcalDeadCellBoundaryEnergyFilter)
+Flag_ecalBadCalibSep2017ListFilter = cms.Path(ecalBadCalibSep2017ListFilter)
```

Comment:
this file is out of sync (ecalBadCalibSep2017ListFilter -> ecalBadCalibFilter is needed

Reply:
Also, this path should be empty by default:
```Flag_ecalBadCalibFilter = cms.Path()```
and then around L66 add
```  phase2_common.toReplaceWith( Flag_ecalBadCalibFilter, cms.Path(ecalBadCalibFilter) )```